saleor/order/tests/test_calculations.py

```diff
@@ -1181,6 +1187,7 @@ def test_fetch_order_data_calls_plugin(
     mock_get_taxes.assert_not_called()
 
 
+@patch("saleor.order.calculations.validate_tax_data")
```

Comment:
Same here. I do not see any check or override regarding this mock

Reply:
I mock the function to skip the validation, since the test use dummy, invalid tax data